### DISCUSSION TITLE
How can i open browser to run ?

### DISCUSSION THREAD

**Discussion by JianJianMa** [Skyvern-AI/skyvern#556] [Q&A] [answered]
When I execute a task, it doesn't open the browser to run, but it seems to run in a headless environment,how can i resolve this

**Comment by suchintan** [comment, marked as answer]
@JianJianMa Are you running Skyvern via Docker or the contributor setup? Unfotunately if it's running through docker it runs in a "headless" environment.

@wintonzheng is planning on launching our task-livestreaming feature inside Docker soon which may help alleviate your concerns!

**Reply by JianJianMa** [reply]
Oh,i see.I'm running in docker; By the way, I have another question. If there is no text on this image or button, it seems that it cannot be recognized to click it

**Reply by nvg990** [reply]
Hello guys @wintonzheng @suchintan, do you have an update on this? Can we run a headless environment in Docker yet?  Thanks for the cool work!
